pkg/webhook/pod/validating/cluster_colocation_profile.go

```diff
@@ -68,6 +69,16 @@ func (h *PodValidatingHandler) clusterColocationProfileValidatingPod(ctx context
 	return allowed, reason, nil
 }
 
+func forbidReservationAnnotations(pod *corev1.Pod) field.ErrorList {
```

Comment:
how about `forbidSpecialAnnotations`

Reply:
+1, and we can maintain a list that with special annotations.

Reply:
👌